config/elasticsearch.yml

```diff
@@ -20,7 +20,7 @@ http:
 
 searchguard:
     ssl.transport:
-        enabled: true
```

Comment:
I don't know if it is possible to not install search-guard-ssl because they packaged it inside search-guard plugin since es5.

 I'll resume tomorrow as it is middle of the night here.

Thank you for reporting bruno